src/cli/config.ts

```diff
@@ -9,8 +9,16 @@ export type Config = {
     game_id: string;
 };
 
+async function readFile (file: string) {
+    try {
+        return await fs.readFile(file, "utf8");
+    } catch (e) {
+        throw new Error(`Configuration file playpass.toml could not be found. Expected at: ${file}`);
```

Comment:
We should filter for `ENOENT` here to prevent showing this "file not found" message for permissions problems or other error types. You can check out `src/cli/auth/cli.ts` for an example of how to do this.